<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flight_inquiry_flightID</class>
 <widget class="QWidget" name="flight_inquiry_flightID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flight Query</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../build-client-Desktop_Qt_5_14_2_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc">
    <normaloff>:/qss/Aqua.qss</normaloff>:/qss/Aqua.qss</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="hint_label">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="flighID_lineEdit">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Confirm_pushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Back_pushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="UserStatus">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="login_pushButton">
       <property name="font">
        <font>
         <family>Yu Gothic UI Semilight</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="flightInfo_label">
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../build-client-Desktop_Qt_5_14_2_MinGW_64_bit-Debug/debug/qmake_qmake_immediate.qrc"/>
 </resources>
 <connections/>
</ui>
